modules/installation-osp-kuryr-known-limitations.adoc

```diff
@@ -5,20 +5,27 @@
 [id="installation-osp-kuryr-known-limitations_{context}"]
 = Known limitations of installing with Kuryr
 
-There are known limitations when using Kuryr SDN:
+There are known limitations when using Kuryr SDN, depending on the
+{rh-openstack} version:
 
 * An Amphora load balancer VM is deployed per OpenShift Service with
```

Comment:
The line before this list says that limitations depend on the OSP version. Which version does this first item apply to?

Reply:
Or if it doesn't break down like that, we can rearrange :)

Reply:
that is what it is explained below